azurerm/resource_arm_container_group.go

```diff
@@ -185,13 +279,13 @@ func resourceArmContainerGroup() *schema.Resource {
 							Elem:     &schema.Schema{Type: schema.TypeString},
 						},
 
-						"volume": {
```

Comment:
As this is a breaking change we are going to have to support both methods here until 2.0